README.md

```diff
@@ -1 +1,33 @@
-# plugin_sublime
\ No newline at end of file
+![Pieces Sublime Plugin](https://camo.githubusercontent.com/69c990240f877927146712d45be2f690085b9e45b4420736aa373917f8e0b2c8/68747470733a2f2f73746f726167652e676f6f676c65617069732e636f6d2f7069656365735f7374617469635f7265736f75726365732f7066645f77696b692f5049454345535f4d41494e5f4c4f474f5f57494b492e706e67)
+
+# Pieces Sublime Plugin
+
+A robust plugin designed to manage your pieces snippets and interact seamlessly with the Pieces OS client.
+
+[Website](https://pieces.app/) | [Pieces OS Documentation](https://docs.pieces.app/)
+
+## Getting Started
+
+Welcome to the Pieces Sublime Plugin. To get started, navigate to `Preferences > Package Settings > Pieces > Keybindings` to check out and modify the keybindings. If you wish to change the llm model you're using or the host, you can do so in the settings.
+
+### Open Asset Command
+
+To open your assets, simply open your command palette and run `Pieces: Open an asset`. Then, select the asset you wish to open.
+
+#### Editor Mode
+
+Enter the editor mode with `command/ctrl + e`. You can then modify your asset and save it using `command/ctrl + s`.
+
+#### Copy an Asset
+
+If you're in an asset sheet, you can quickly copy the asset content without selecting it using `command/ctrl + c`.
+
+### Copilot Ask
+
+Need the copilot to comment, fix a bug, or add a doc string? Simply go to the command palette and run `Pieces: Ask Copilot`.
```

Comment:
do we have this feature in the sublime plugin?

Reply:
https://github.com/pieces-app/plugin_sublime/pull/21

Just the refactor command

Reply:
If we don't have this feature, we should strike this bullet or make it more explicit about what the feature is. "Refactor Code" or something.

Reply:
& how does this feature work exactly? Is there a demo video somewhere I can check out?

Reply:
Would move this one closer to the top of the document.